MCR-1525 update hibernate version to current version

kkrebs opened the pull request.

kkrebs committed:
MCR-1525 update hibernate version to current version

MyCoRe-Travis commented:
Something gone wrong on 07dc20164ececceb5833ef462f083b617c5dd057! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/1310-pull_request-234

MyCoRe-Travis commented:
Something gone wrong on 24e7ae4ab7cc416aae314e8b214cc3be53788407! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/1313-pull_request-234

MyCoRe-Travis commented:
Something gone wrong on 90227e940d906064e988f04ef4e512e9ca877eed! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/1315-pull_request-234

yagee-de committed:
MCR-1557 Persistence unit for tests is component-name

Hibernate >5.2.3 requires the persistence units may only be configured once
in a classpath. So we name every persistence unit under test after its component name,
e.g. "mycore-base".
